apps/myaccount/src/components/account-recovery/account-recovery.tsx

```diff
@@ -187,13 +210,26 @@ export const AccountRecoveryComponent: FunctionComponent<AccountRecoveryProps> =
                             featureConfig?.security,
                             AppConstants.FEATURE_DICTIONARY.get("SECURITY_ACCOUNT_RECOVERY_EMAIL_RECOVERY")
                         ) &&
-                        (isNotificationRecoveryEnabled || isUsernameRecoveryEnabled) ? (
+                        (isNotificationRecoveryEmailLinkEnabled || isUsernameRecoveryEnabled) ? (
                                 <EmailRecovery
                                     onAlertFired={ onAlertFired }
                                     data-testid={ `${testId}-settings-section-email-recovery` }
                                 />
                             ) : null }
                     </List.Item>
+                    <List.Item className="inner-list-item">
+                        { hasRequiredScopes(
```

Comment:
Let's use the `useRequiredScopes` hook instead. It is more convenient and compatible with unit tests.

Reply:
Addressed with https://github.com/wso2/identity-apps/pull/6418/commits/c189193598ea63ba97e745308cac1b8b89ce7403